docs/conf.py

```diff
@@ -71,12 +71,8 @@ def setup(app):
 sys.path.append(os.path.abspath("./_ext"))
 
 extensions = [
-    'sphinxcontrib.httpdomain',
-    'sphinx.ext.extlinks',
-    'sphinx.ext.ifconfig',
-    'sphinx_tabs.tabs',
-    'sphinx_copybutton',
-    'validating_code_block'
+    'sphinxcontrib.httpdomain', 'sphinx.ext.extlinks', 'sphinx.ext.ifconfig', 'sphinx_tabs.tabs',
```

Comment:
grrr, over-opinionated linter - thats the second time today for me https://github.com/translate/translate/pull/4131#issuecomment-697314968

Reply:
Yeah. I also preferred my way but what can you do. It should be fixed in e6e851a749ca8e8ec06baec2a430d52ebd2d5345